examples/actions.js

```diff
@@ -31,9 +31,9 @@ app.action({
     from: intentA,
     to: intentB,
     if: (slots, attrs) => attrs.date, // Note: date should be validated here
-    fail: (slots, attrs) => 'Sorry, your command is invalid'
+    fail: () => 'Sorry, your command is invalid'
```

Comment:
I would like to keep these props in `/examples` even if unused just to make sure developers know it is available
- put these parameters back and exclude `/examples` from linting
- or would it be possible to use different eslint config for `/examples`? (with the changed rule `no-unused-vars`